controllers/apply/reaching-communities/processor.test.js

```diff
@@ -1,150 +0,0 @@
-/* eslint-env jest */
-'use strict';
-const fs = require('fs');
-const path = require('path');
-const yaml = require('js-yaml');
-const nodemailer = require('nodemailer');
-const { get, map } = require('lodash');
-const jsdom = require('jsdom');
-const { JSDOM } = jsdom;
-
-const { flattenFormData, stepsWithValues } = require('../form-router/helpers');
-const form = require('./form-model');
-const processor = require('./processor');
-
-function mockProcessor() {
-    const langData = fs.readFileSync(
-        path.resolve(__dirname, '../../../config/locales/en.yml'),
-        'utf-8'
-    );
-    const enCopy = yaml.safeLoad(langData);
-    const formCopy = get(enCopy, form.lang);
-
-    const mockFormData = {
-        'step-1': {
-            'your-idea': 'Test idea'
-        },
-        'step-2': {
-            'location': 'South West',
-            'project-location': 'London'
-        },
-        'step-3': {
-            'organisation-name': 'Test Organisation',
-            'additional-organisations': ''
-        },
-        'step-4': {
-            'first-name': 'Example',
-            'last-name': 'Person',
-            'email': 'example@example.com',
-            'phone-number': '03454102030'
-        }
-    };
-
-    const mockTransport = nodemailer.createTransport({
-        jsonTransport: true
-    });
-
-    const processorOptions = {
-        data: flattenFormData(mockFormData),
-        copy: formCopy,
-        stepsWithValues: stepsWithValues(form.steps, mockFormData)
-    };
-
-    return processor(processorOptions, mockTransport);
-}
-
-describe('reaching communities processor', () => {
```

Comment:
I feel _OK_ about removing this test as it's very fake i.e. mocks far too many things and requires a client-side build to work which is an anti-pattern but appreciate it does add _some_ value to open to looking at other ways to test this before merging. 💭